src/test/java/BaseTest.java

```diff
@@ -1,11 +1,57 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.*;
+
+import java.time.Duration;
 
 public class BaseTest {
+    public WebDriver driver;
 
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
-}
\ No newline at end of file
+
+    @BeforeMethod
+    @Parameters("baseUrl")
+    public void launchBrowser(String baseUrl){
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        // version fix
+        //System.setProperty("webdriver.chrome.driver", "chromedriver-mac-arm64/chromedriver");
+
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        driver.get(baseUrl);
+    }
+
+    //Helper Methods
+    public void loginToKoel() {
+        WebElement loginBtn =  driver.findElement(By.cssSelector("button[type='submit']"));
+        loginBtn.click();
+        //Thread.sleep(5000);
+    }
+
+    public void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys(password);
+    }
+
+    public void provideEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("input[type='email']"));
+        emailField.clear();
+        emailField.sendKeys(email);
+    }
```

Comment:
Page Object Model (POM): recommended to associate the methods with the corresponding page class where they are relevant. In the POM, each page or section of a web application is represented by a separate class. These classes encapsulate the elements and actions specific to that page or section.

Move these methods to the corresponding class according to the POM.

Reply:
That was refactored, ill take a look at the versions. Thank you Alex